Add BoldSign custom connector

mohanravi07 opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

mohanravi07 committed:
add BoldSign custom connector

mohanravi07 committed:
update description

mohanravi07 committed:
add region in readme

mohanravi07 commented:
@microsoft-github-policy-service agree company="Syncfusion, Inc."

vmanoharas commented on certified-connectors/BoldSign/README.md:
```diff
@@ -0,0 +1,31 @@
+# Introduction
+
```
Please update the readme.md file as per the template below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

mohanravi07 replied:
Updated readme as per template

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "events": {
+                  "type": "string",
+                  "description": "events",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "environment": {
+                  "type": "string",
+                  "description": "environment",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "isActive": {
+                  "type": "boolean",
+                  "description": "isActive",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "webhookType": {
+                  "type": "string",
+                  "description": "webhookType",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas commented on certified-connectors/BoldSign/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,1536 @@
+                "developerAppId": {
+                  "type": "string",
+                  "description": "developerAppId",
+                  "title": "",
```
Please use proper title instead of empty value.

mohanravi07 replied:
Added proper title.

vmanoharas reviewed (commented):
Hello @mohanravi07

Please resolve review comments.

mohanravi07 committed:
add descriptions

mohanravi07 committed:
add dynamic auto reminder support

mohanravi07 committed:
update readme

mohanravi07 commented:
Hi @vmanoharas, Sorry for the delay. Somehow, I missed this. I have now addressed the comments. Please take another look at this and let me know if you have any concerns.

vmanoharas reviewed (approved):
Hello @mohanravi07 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to Connector certification portal](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in [Connector certification portal](https://connectorcertification.azurewebsites.net/) . Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in [Connector certification portal](https://connectorcertification.azurewebsites.net/), please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

mohanravi07 commented:
Hi @vmanoharas,

We tried to submit the connector using the link below, but it is throwing an error. Can you please guide us on how to proceed next?

![image](https://github.com/microsoft/PowerPlatformConnectors/assets/29397671/6c6b2106-3c83-4666-9aee-0745191426cb)

vmanoharas commented:
Hello @mohanravi07,

Our connector certification process has been changed, kindly follow below instructions. 

We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team
